rfc-drafts/must_not_await_lint.md

```diff
@@ -16,26 +16,26 @@ The big reason for including a lint like this is because under the hood the comp
 
 Provide a lint that can be attached to structs to let the compiler know that this struct can not be held across an await boundary.
 
-```rust
+```rust,ignore
 #[must_not_await = "Your error message here"]
 struct MyStruct {}
-```
+```ignore
```

Comment:
I don't think these "ignore" tags on the closing backticks is correct (at least none of my markdown parsers understands this)  I'm going to open a PR to fix this